package.json

```diff
@@ -39,10 +39,12 @@
     "i18next-browser-languagedetector": "^6.1.2",
     "i18next-http-backend": "^1.3.1",
     "identity-obj-proxy": "^3.0.0",
+    "install": "^0.13.0",
```

Comment:
please run `npm uninstall install` to remove this and commit & push the package.json and package-lock.json files without it